# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 10b30c8d4b86
# Full Hash: 10b30c8d4b860b7d39de8d46fd186474407626b2
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-05-14 21:58:21
# Regressor Bug: 1708768
# File Overlap Count: 1
# Description:
#   Bug 1708768 - Create a pref to defer gfxPlatformFontList initialization until first use, and enable this on macOS. r=lsalzman
#   
#   This gives the RegisterFonts thread more time to complete its work, so that
#   the main thread doesn't have to wait for it in InitFontList.
#   
# ==============================================================================

diff -r c03c7df7cd61 -r 10b30c8d4b86 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri May 14 18:05:44 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri May 14 18:05:44 2021 +0000
@@ -4610,6 +4610,15 @@
   value: true
   mirror: once
 
+- name: gfx.font-list.lazy-init.enabled
+  type: bool
+#if defined(XP_MACOSX)
+  value: true
+#else
+  value: false
+#endif
+  mirror: once
+
 # Whether to load fonts (e.g. Twemoji Mozilla) bundled with the application:
 #  -1 - Auto behavior based on OS version (currently, disables loading on
 #       "low-memory" Android devices)
